composer.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "goalgorilla/shariff",
+    "description": "Shariff enables website users to share their favorite content without compromising their privacy.",
+    "type": "npm-asset",
+    "minimum-stability": "dev",
+    "require": {}
```

```suggestion
    "require": {},
    "replace": {
        "npm-asset/shariff": "self.version"
    } 
```

As per [the Composer documentation](https://getcomposer.org/doc/04-schema.md#replace).

Lets ensure that we can use this new version already even if some package still requires the old one (this also prevents both packages from being installed at the same time).